### ISSUE TITLE
Verification of Plots and Equations

### ISSUE THREAD

**Issue by aditeyabaral** [aditeyabaral/GPython#1]
Some equations do not support plotting. These equations are of the form y <sup>n</sup> = f(x).
It would be great if these plots can be added somehow!

**Comment by ajayjha5899** [comment]
Can you please explain y<sup>n</sup> = f(x)
As f(x) must be a function in x.

**Comment by ajayjha5899** [comment]
```sh
$ python3 Plotter.py 
Enter function : (3*x+5)**0.1
```
This is working fine.
[P.S- Please correct me if I've misunderstood y<sup>n</sup> = f(x).]

**Comment by aditeyabaral** [comment]
Hello, 
If you take the example of a circle at Origin given by x <sup>2</sup> + y<sup>2</sup> = r<sup>2</sup>, then the application requires an input as a function of y, hence it would turn out to be y = sqrt(r<sup>2</sup> - x<sup>2</sup>). Plotting this would only return an incomplete circle. 
This issue would arise for all even powers due to the nature of the function (as well as 3D plots). It would be great to find a workaround, or maybe even a different algorithm that handles all cases!

**Comment by aditeyabaral** [comment]
Just an addition to this - 
This can be obtained using the `numexpr` library. It can be used to evaluate expressions and can be used to remove the `eval` function used in the current implementation. Check [here](https://github.com/pydata/numexpr) for more details.

As far as I can see, the only generic approach is to evaluate the LHS and RHS separately or in parallel (if possible) and then equate and pick the values that do satisfy. However any other workaround is appreciated.

**Comment by Chakita** [comment]
I'd like to take up this issue, and replace eval with numexpr

**Comment by aditeyabaral** [comment]
Alright sure

**Comment by ritwikchakraborty123** [comment]
@aditeyabaral  I want to contribute

**Comment by aditeyabaral** [comment]
> 
> 
> @aditeyabaral I want to contribute

@ritwikchakraborty123 this issue is currently being handled by @Chakita, please contribute to another issue
